apps/remix-ide/src/app/plugins/solcoderAI.tsx

```diff
@@ -151,7 +147,6 @@ export class SolCoder extends Plugin {
       ).json()
 
       if ("error" in result){
-        this.call('terminal', 'log', { type: 'aitypewriterwarning', value: result.error }) 
```

Comment:
you removed the error too?

Reply:
Yeah, this is the silly "doc < etc." error for code completion. Since we are monitoring the health, no need to spam the user. I could add it to the console